integration_tests/CMakeLists.txt

```diff
@@ -509,7 +509,7 @@ RUN(NAME test_pkg_lnn_01        LABELS cpython llvm)
 RUN(NAME test_pkg_lnn_02        LABELS cpython llvm)
 
 RUN(NAME generics_01         LABELS cpython llvm c)
-RUN(NAME generics_02         LABELS cpython llvm c)
+# RUN(NAME generics_02         LABELS cpython llvm c)
```

Comment:
I temporarily commented this out.

Reply:
I made the PR a draft until we resolve it. First let's get everything else working, then we'll fix this.